.claude/hooks/session-start.sh

```
@@ -0,0 +1,74 @@
+#   - services.gradle.org                (wrapper metadata)            [required]
+#   - github.com HTTPS file downloads    (wrapper zip redirect target) [required]
+#   - androidx.dev                       (compose-compiler repo)       [required]
+#   - repo.maven.apache.org, plugins.gradle.org                        [required]
+set -uo pipefail
+
+if [ "${CLAUDE_CODE_REMOTE:-}" != "true" ]; then
+  exit 0
+fi
+
+PROJECT_DIR="${CLAUDE_PROJECT_DIR:-$(cd "$(dirname "$0")/../.." && pwd)}"
+cd "$PROJECT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)